src/SetCookie.php

```diff
@@ -90,7 +98,7 @@ public static function thatStaysForever(
     ) : SetCookie {
         $expiresInFiveYear = time() + 5 * 365 * 3600 * 24;
 
-        return new static($name, $value, $expiresInFiveYear, $path, $domain, $secure, $httpOnly, $sameSite);
+        return new self($name, $value, $expiresInFiveYear, $path, $domain, $secure, $httpOnly, $sameSite);
```

Comment:
Let's keep this static, there's no reason to change this? both will point to same because it's `final`, so it's just unneeded changes? If this class would be extendable then `static` is often prefered.

Reply:
Again, some analysis tools will yell at it.